File: java/advanced/APIDemo/app/src/main/java/com/google/android/snippets/AdmobTestAds.java
package com.google.android.snippets;

import com.google.android.gms.ads.MobileAds;
import com.google.android.gms.ads.RequestConfiguration;
import java.util.Arrays;

/** This snippet shows how to initialize test ads. */
final class AdmobTestAds {

  private AdmobTestAds() {}

  public static void initializeTestAds() {
    // [START initialize_test_ads]
    java.util.List<String> testDeviceIds = Arrays.asList("33BE2250B43518CCDA7DE426D04EE231");
    RequestConfiguration configuration =
        new RequestConfiguration.Builder().setTestDeviceIds(testDeviceIds).build();
    MobileAds.setRequestConfiguration(configuration);
    // [END initialize_test_ads]
  }
}

File: kotlin/advanced/APIDemo/app/src/main/java/com/google/android/gms/example/apidemo/snippets/AdmobTestAds.kt
package com.google.android.gms.example.apidemo

import com.google.android.gms.ads.MobileAds
import com.google.android.gms.ads.RequestConfiguration
import java.util.Arrays

/** This snippet shows how to initialize test ads. */
class AdmobTestAds {

  fun initializeTestAds() {
    // [START initialize_test_ads]
    val testDeviceIds = Arrays.asList("33BE2250B43518CCDA7DE426D04EE231")
    val configuration = RequestConfiguration.Builder().setTestDeviceIds(testDeviceIds).build()
    MobileAds.setRequestConfiguration(configuration)
    // [END initialize_test_ads]
  }
}
